### PR TITLE
Fix GitHub pjax issues

### LINKED ISSUE #3: Doesn’t work when the “Issues” link in the top bar is clicked
The “Issues” link on the right-hand sidebar (which points to e.g. `https://github.com/sindresorhus/github-issues-all/issues`) works fine, but when the “Issues” link in the top bar (which points to e.g. `https://github.com/sindresorhus/github-issues-all/issues?q=is%3Aopen+is%3Aissue` instead) is clicked, i.e. the one with the blue background in this screenshot:

…then the plugin doesn’t take effect.

### LINKED ISSUE #4: No longer redirects on Chrome 46
This does not seem to work any longer. I'm running Chrome 46.0.2490.86 (64-bit). (No idea when it stopped working, as I just installed it.) But the redirect to `issues?q=is:open` doesn't occur.

When I get some time, I'll try to fix this. Opening an issue for future reference...

### PR DESCRIPTION
**Pull request by jeffslofish**
I believe that #3 and #4 are caused by GitHub using pjax. This pull request should close both these issues. 

fixes #3, fixes #4

### CODE REVIEW

File: background.js [hunk 1]
```diff
@@ -2,16 +2,28 @@
 
 chrome.webRequest.onBeforeRequest.addListener(function (details) {
 	if (details.method === 'GET') {
+		const isOpenQuery = 'q=is:open';
+		const pjaxQueryStart = details.url.search(/\?_pjax=%23js-repo-pjax-container/);
+
+		let newUrl;
+		if (pjaxQueryStart !== -1) {
+			newUrl = details.url.substr(0, pjaxQueryStart + 1) + isOpenQuery;
```
Comment (sindresorhus) [inline background.js]:
```suggestion
			newUrl = details.url.slice(0, pjaxQueryStart + 1) + isOpenQuery;
```

I prefer `.slice` for consistency.

File: background.js [hunk 2]
```diff
@@ -2,16 +2,28 @@
 
 chrome.webRequest.onBeforeRequest.addListener(function (details) {
 	if (details.method === 'GET') {
+		const isOpenQuery = 'q=is:open';
+		const pjaxQueryStart = details.url.search(/\?_pjax=%23js-repo-pjax-container/);
```
Comment (sindresorhus) [inline background.js]: Can you use `new URL` to properly parse out the query string?

**Comment by sindresorhus** [comment]
I cannot get it to work with the latest commit. It never shows the PRs.

**Comment by jeffslofish** [comment]
It is weird...for me GitHub is not triggering pjax requests when I click the issues tab anymore, and because of this, your original code at https://github.com/sindresorhus/show-all-github-issues is actually working for me when I load that as an extension.

However, if your code is not working for you, then maybe somehow pjax requests are being triggered on your end but not on mine. If that is the case then it is difficult for me to debug the pjax related code, which I suspect I broke in my latest change in the pull request and that is why my pull request no longer works for you, but it does for me. I'm not sure what to do at this point...

**Comment by sindresorhus** [comment]
Turns out it's caused by a conflict with https://github.com/sindresorhus/refined-github. When I disabled Refined GitHub it started working.

**Pull request merged**